### PR TITLE
Improve generator script

### LINKED ISSUE #494: How to run readmeGenerator?
Hi. I found a brilliant idea to made generator instead of manuall building Readme. 👍 

I want to improe it a little bit, but can't run.

I open `.github/ReadmeGenerator.xcodeproj`

Then run program and got this warning:

Screenshot 2019-12-28 at 23 10 27
I don't know, where is your name comes from, look like so absolute path settled, but 

`grep -iR "Users/serhii-londar/" ./**` doesn't shoe any matches.

If I continue to run the project, got the error output:

```swift
Start
Error Domain=NSCocoaErrorDomain Code=260 "The file “applications.json” couldn’t be opened because there is no such file." UserInfo={NSFilePath=/Users/pk/Library/Developer/Xcode/DerivedData/ReadmeGenerator-fmeoviztnomraycobfgvbwocervo/Build/Products/Debug/applications.json, NSUnderlyingError=0x103851400 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
Program ended with exit code: 0
```

Please, help to run `main.swift` and I would be glad to improve this script! 👍

### PR DESCRIPTION
**Pull request by skywinder**
this will fix #494:
- fix userscheme
- update generator to support any kind of paths (absolute and relative
  paths) 
- made xcodeproj runnnuble by anyone

### CODE REVIEW

**Comment by skywinder** [comment]
both methods works: 
via running xcode proj and `swift ./.github/main.swift` from the root

**Comment by skywinder** [comment]
Are you sure, that it will fail Travis? looks green in this PR ⬇️

Screenshot 2020-01-01 at 06 44 33

File: .github/main.swift [hunk 1]
```diff
@@ -213,8 +213,16 @@ class ReadmeGenerator {
     func generateReadme() {
         print("Start")
         do {
-            let applicationsData = try Data(contentsOf: URL(fileURLWithPath: FilePaths.applications.rawValue))
-            let categoriesData = try Data(contentsOf: URL(fileURLWithPath: FilePaths.categories.rawValue))
+            // get current file path:
+            let thisFilePath:String = #file
+            var url = URL(fileURLWithPath: thisFilePath)
+
+            //cd ../ to the root folder: (delete `.github/main.swift`
```
Comment (skywinder) [inline .github/main.swift]: oh, if you talking about this comment- no worry. it's more tricky. it goes always from the locations, where the file is stored. it's independent from the path, where you start it from! 🆗

**Comment by serhii-londar** [comment]
@skywinder Thank you!

**Pull request merged**